src/components/LogIn.jsx

```diff
@@ -1,55 +1,69 @@
-
-import {useState} from 'react';
+/* eslint-disable valid-jsdoc */
```

Comment:
See comment above. Once you disable this rule for the project, you can delete these file-specific comments